bot.py

```diff
@@ -62,6 +62,28 @@ def get_insult(name):
     return random.choice(insults)
 
 
+#added list of praises
+def get_praise(name):
+    insults = [
```

Comment:
should be praises

Reply:
but it was insult so i made it praise to match :(

Reply:
look at the variable bro

Reply:
line 67

Reply:
o that variable
yes 
hmmphh fine